components/automate-cs-ocid/habitat/hooks/run

```diff
@@ -11,6 +11,11 @@ export CHEF_SECRETS_DATA=$(cat config/private-chef-secrets.json)
 
 export DATABASE_URL="{{cfg.DATABASE_URL}}"
 
+WEBUI_SRC_PATH="/hab/svc/automate-cs-oc-erchef/data/webui_priv.pem"
+WEBUI_KEY=`sed ':a;N;$!ba;s/\n/\\n/g' $WEBUI_SRC_PATH`
```

Comment:
@RajeshPaul38 From the code it's not clear what is happening here. We can add a descriptive comment.

Reply:
sed is reading the content from this file and assigning to the variable. It's used in chef-server.